packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -403,6 +404,36 @@ describe('Picker', function () {
 
       expect(document.activeElement).toBe(listbox);
     });
+
+    it('scrolls the selected item into view on menu open', function () {
+      let scrollToSpy = jest.fn();
+      let virtualizerMock = jest.spyOn(Virtualizer.prototype, 'scrollToItem').mockImplementationOnce(scrollToSpy);
```

Comment:
Best I could come up with, everything else I tried to make the Picker dropdown actually bring "Four" into view didn't quite work so instead I check that the scrollToSpy is being called with the correct key. Open to suggestions on how to actually make the Picker dropdown perform properly...

Reply:
Would it be better to do a table test instead of a picker test?

Reply:
I opted for a picker test since that was the component that caught the issue w/ the original approach.